contracts/package.json

```diff
@@ -38,7 +36,7 @@
     "eslint-plugin-prettier": "^3.4.0",
     "eslint-plugin-promise": "^5.1.0",
     "ethereum-waffle": "^3.0.0",
-    "ethers": "^5.5.1",
+    "ethers": "5.5.4",
```

Comment:
`ethers@5.6.0` makes everything very sad, explicitly pinned `5.5.4` until someone can investigate upgrade path, likely in https://github.com/jzaki/bls-wallet/issues/160